packages/diagrams/frontend/sirius-components-diagrams/src/renderer/edge/CommonEdge.types.ts

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+export interface CommonEdgeProps {
```

Comment:
I propose something like
`
export type CommonEdgeProps<T = any> = {
  edgeCenterX: number;
  edgeCenterY: number;
  svgPathString: string;
} & EdgeProps<T>;
` to still use EdgeProps, what do you think ?

Reply:
Well better.
Done